### ISSUE TITLE
AssemblyName is how you name your Mac Catalyst app

### ISSUE THREAD

**Issue by davidbritch** [dotnet/docs-maui#1843]
### Type of issue

Outdated article

### Description

From Maddy: AssemblyName is how you name your Mac Catalyst app - Title isn't respected. This is an Apple bug.

### Page URL

https://learn.microsoft.com/en-us/dotnet/maui/deployment/visual-studio-properties

### Content source URL

https://github.com/dotnet/docs-maui/blob/main/docs/deployment/visual-studio-properties.md

### Document Version Independent Id

a0ecfb0a-e544-7118-d3bf-736bb4d3eb09

### Article author

davidbritch

**Comment by jonathanpeppers** [comment]
@rolfbjarne we noticed this behavior here:
* https://discord.com/channels/732297728826277939/732297916680765551/1238161549496160317

It looks like you could also set `$(_AppBundleName)`, but we shouldn't use a private MSBuild property:

* https://github.com/xamarin/xamarin-macios/blob/3a4fe453f7574796308431e17a1e7ad9f5479cce/msbuild/Xamarin.Shared/Xamarin.Shared.props#L310

Should we come up with a public MSBuild property for this? And then .NET MAUI's `$(ApplicationTitle)` could also set it?

**Comment by rolfbjarne** [comment]
Yes, this looks like something we can improve in xamarin-macios, so I've moved the issue there.
